cmd/peek-submission/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/courses/assignments/submissions"
+	"github.com/edulinq/autograder/internal/api/server"
+	"github.com/edulinq/autograder/internal/common"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+var args struct {
+	config.ConfigArgs
+	TargetEmail      string `help:"Email of the user to fetch." arg:""`
+	CourseID         string `help:"ID of the course." arg:""`
+	AssignmentID     string `help:"ID of the assignment." arg:""`
+	TargetSubmission string `help:"ID of the submission. Defaults to the latest submission." arg:"" optional:""`
+	ShortForm        bool   `help:"Use short form output." long:"short-form"`
```

Comment:
I would change it to just something like `--short`.

As a side note, doesn't Kong get that same long form from the field's name
(I think it observes casing).